.gitignore

```diff
@@ -15,3 +15,23 @@ mingw_build/
 .vscode
 cmake-build-debug/
 cmake-build-release/
+
+resources/*.AF
```

Comment:
What does this .gitignore change do ?

Reply:
This is where debug build expects resources to be found, but guess it should not be part of this pull request.